# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/cookie-store/__dir__.ini
# Commit: 9c8b6e35bf3f
# Full Hash: 9c8b6e35bf3f9916e114ced0473be4b778c7bab1
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-11 09:21:34
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 1 - CookieStore API - WebIDL, r=smaug,edgul
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215144
# ==============================================================================

diff -r a9b3b52f33a5 -r 9c8b6e35bf3f testing/web-platform/meta/cookie-store/__dir__.ini
--- a/testing/web-platform/meta/cookie-store/__dir__.ini	Tue Sep 10 16:21:33 2024 +0000
+++ b/testing/web-platform/meta/cookie-store/__dir__.ini	Tue Sep 10 16:31:38 2024 +0000
@@ -1,3 +1,4 @@
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1475599
+prefs: [dom.cookieStore.enabled:true]
 implementation-status: backlog
 lsan-allowed: [Alloc, Malloc, Then, mozilla::BasePrincipal::CreateContentPrincipal, mozilla::dom::DocGroup::Create, mozilla::dom::ServiceWorkerJobQueue::RunJob, mozilla::dom::ServiceWorkerManager::Unregister, mozilla::dom::ServiceWorkerRegistrationMainThread::Unregister, mozilla::dom::UnregisterCallback::UnregisterCallback, mozilla::net::nsStandardURL::TemplatedMutator, operator]